main.go

```diff
@@ -360,3 +363,23 @@ func handleSignals(ctx context.Context) error {
 		}
 	}
 }
+
+func createC2PA(cli *config.Cli) *c2pa.C2PA {
```

Comment:
Shall we have this also return an error to be more Go-ish? Then we can log in one place if it fails and unit test the failure cases

Reply:
Good idea, updated in 098b892bd46f01574387dbcde8e0ca79403012f0